src/interface/src/app/home/welcome/welcome.component.scss

```diff
@@ -26,8 +37,14 @@ p {
 .flex {
   display: flex;
   align-items: flex-start;
+  gap:20px;
```

Comment:
This makes me curious: do we enforce linting conventions for CSS/SCSS? Should we? 
I'm fine either way, just was surprised to see no space here.